### PR TITLE
lesson_02_flow_control problems

### PR DESCRIPTION
**Pull request by sendeand**
@kuszcjan 
Please check my commits for lesson02 - Flow Control.

### CODE REVIEW

**Comment by kuszcjan** [comment]
Close enough :)

File: students/sendecki_andrzej/lesson_02_flow_control/lesson3-chocolate_bar.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+print("Enter chocolate heigth:")
+m = int(input())
+
+print("Enter desired chocolate pieces:")
+k = int(input())
+
+#1.
+if k > m*n :
+    print("NO")
```
[the remark this answers is not in the document]
Comment (sendeand) [reply students/sendecki_andrzej/lesson_02_flow_control/lesson3-chocolate_bar.py]: I agree but that were the requirements.

File: students/sendecki_andrzej/lesson_02_flow_control/lesson3-knight_move.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+c2x = int(input())
+if (c2x < 1 ) or (c2x > 8):
+    print("Error: number out of range")
+    sys.exit()
+
+print("y:")
+c2y = int(input())
+if (c2y < 1 ) or (c2y > 8):
+    print("Error: number out of range")
+    sys.exit()
```
[the remark this answers is not in the document]
Comment (sendeand) [reply students/sendecki_andrzej/lesson_02_flow_control/lesson3-knight_move.py]: In this very case it is convenient - I limit the input a the beginning thus I do not have put extra checks later.

File: students/sendecki_andrzej/lesson_02_flow_control/lesson3-leap_year.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+print("Enter the year:")
+year=input()
```
[the remark this answers is not in the document]
Comment (sendeand) [reply students/sendecki_andrzej/lesson_02_flow_control/lesson3-leap_year.py]: Will keep that in mind, thx.

**Pull request merged**